DirtyLogger/DirtyLogger.hpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * DirtyLogger.hpp
+ *
+ *  Created on: Dec 31, 2023
+ *      Author: jurcz
+ */
+
+#ifndef DIRTYLOGGER_DIRTYLOGGER_HPP_
+#define DIRTYLOGGER_DIRTYLOGGER_HPP_
+#include "fatfs.h"
+
+
+class DirtyLogger{
+public:
+	DirtyLogger(uint8_t *retSD, char *SDPath, FATFS *_SDFatFS, FIL *SDFile);
+	void Init(void);
+	void Log(const char* message, bool include_timestamp);
+	void StartTimestamp(void);
+private:
+	uint8_t wtext[_MAX_SS]; /* File write buffer */
```

Comment:
why not name it "fileWriteBuffer" instead of writing comment? :D You can consider renaming it.

ofc it can also stay as is. It's your repo. I don't want to be that annoying guy who makes irrelevant comments in the review and wants everything his way.

Reply:
Not used in thge code so I will delete it